src/automations/components/AutomationActionDescriptionResumeFlowRun.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <p>Resume a flow run</p>
```

Comment:
We try to use some more specific language when we're inferring based on the trigger. For suspend and cancel we say "Suspend flow run inferred from the triggering event". Might be good to be consistent here. 

Alternatively, you could update the "AutomationActionDescriptionSuspendCancelFlowRun.vue" component to handle resume as well. But I kinda like the simplicity of this static component.

Reply:
Oh, thanks for the heads-up!